lib/linear_gauge/linear_gauge_painter.dart

```diff
@@ -388,10 +399,18 @@ class RenderLinearGauge extends RenderBox {
 
     paragraph.layout(ui.ParagraphConstraints(width: labelSize.width));
 
-    canvas.drawParagraph(
-        paragraph,
-        Offset((list[0].dx - (labelSize.width / 2)),
-            list[0].dy + getPrimaryRulersHeight + getLabelTopMargin));
+    if (invertLabels) {
+      canvas.drawParagraph(
+          paragraph,
+          // Offset(0, -(19 + getLinearGaugeBoxDecoration.height)));
+          Offset((list[0].dx - (labelSize.width / 2)),
+              -(getPrimaryRulersHeight + labelSize.height - 2)));
+    } else {
+      canvas.drawParagraph(
```

Comment:
Can you calculate offset using the if condition? and use `canvas.drawParagraph` once

Reply:
`Offset labelOffsets;
    if (invertLabels) {
      labelOffsets = Offset(
        (list[0].dx - (labelSize.width / 2)),
        -(getPrimaryRulersHeight + labelSize.height - 2),
      );
    } else {
      labelOffsets = Offset(
        (list[0].dx - (labelSize.width / 2)),
        (list[0].dy + getPrimaryRulersHeight + getLabelTopMargin),
      );
    }
    canvas.drawParagraph(
      paragraph,
      labelOffsets,
    );`